charts/argo/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "v2.4.3"
+appVersion: "v2.5.0-rc8"
```

Comment:
Why rc? not ready for major release?

Reply:
The latest release is still an RC release at rc10: https://github.com/argoproj/argo/releases